README.md

```diff
@@ -8,4 +8,7 @@ Okay, done.
 
 ### now making more changes here
 
-## and even more lines are added
\ No newline at end of file
+## and even more lines are added
+
+Steps to add feature
+1. Open the index. html in a browser
```

Comment:
this does not look like a first step, are you sure this is the first step in the process?